var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -70,6 +70,30 @@ class SuiteSparse(Package):
 
     conflicts('%gcc@:4.8', when='@5.2.0:', msg='gcc version must be at least 4.9 for suite-sparse@5.2.0:')
 
+    def symbol_suffix_blas(self, spec, args):
+        """When using BLAS with a special symbol suffix we use defines to
+        replace blas symbols, e.g. dgemm_ becomes dgemm_64_ when
+        symbol_suffix=64_."""
+
+        # Currently only OpenBLAS does this.
+        if not spec.satisfies('^openblas'):
+            return
+
+        suffix = spec['openblas'].variants['symbol_suffix'].value
+        if suffix == 'none':
+            return
+
+        symbols = (
+            'dtrsv_', 'dgemv_', 'dtrsm_', 'dgemm_', 'dsyrk_', 'dger_', 'dscal_',
+            'dpotrf_', 'ztrsv_', 'zgemv_', 'ztrsm_', 'zgemm_', 'zherk_',
+            'zgeru_', 'zscal_', 'zpotrf_',
+            'dnrm2_', 'dlarf_', 'dlarfg_', 'dlarft_', 'dlarfb_', 'dznrm2_',
+            'zlarf_', 'zlarfg_', 'zlarft_', 'zlarfb_'
+        )
+
+        for symbol in symbols:
+            args.append('CFLAGS+=-D{0}={1}{2}'.format(symbol, symbol, suffix))
```

Comment:
Would other dependents of blas need this as well?

Reply:
Most likely yes, but I'm not planning to create more of these PRs. `suite-sparse` is used as a solver in Julia's SparseArrays stdlib, and Julia (or rather libblastrampoline) prefers `64_` symbol suffixes for ilp64 BLAS. Since my goal with the Julia PR is to follow the official build as closely as possible, I have to do this.